doc/connectivity/usb/host/usbip.rst

```diff
@@ -0,0 +1,75 @@
+.. _usbip:
+
+USB/IP protocol support
+#######################
+
+Overview
+********
+
+New USB support includes initial support for the USB/IP protocol. It is still
+under development and is currently limited to supporting only one device
+connected to the host controller being exported.
+
+USB/IP uses TCP/IP. Both of the underlying connectivity stacks, USB and
+networking, require significant memory resources, which must be considered when
+choosing a platform.
+
+In the USB/IP protocol, a server exports the USB devices and a client imports
+them. USB/IP support in the Zephyr RTOS implements server functionality and
+exports a device connected to a host controller on a device running the Zephyr
+RTOS. A client, typically running the Linux kernel, imports this device. The
+USB/IP protocol is described in `USB/IP protocol documentation`_.
+
+To use USB/IP support, make sure the required modules are loaded on the client side.
+
+.. code-block:: console
+
+   modprobe vhci_hcd
+   modprobe usbip-core
+   modprobe usbip-host
+
+There are a few basic commands for everyday use. To list exported USB devices,
```

Comment:
worth mentioning packages to get for ubuntu here so users don't have to search for why the command is not found?